cli/command/image/remove_test.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 func TestNewRemoveCommandAlias(t *testing.T) {
-	cmd := newRemoveCommand(test.NewFakeCli(&fakeClient{}, new(bytes.Buffer)))
+	cmd := newRemoveCommand(test.NewFakeCliWithOutput(&fakeClient{}, new(bytes.Buffer)))
```

Comment:
Could just be `NewFakeCli`, but I know you're not trying to exhaustively convert these.